### PR TITLE
Add jest-environment-jsdom-fifteen recommendation

### PR DESCRIPTION
**Pull request by trurl-master**
Fix for https://github.com/testing-library/react-testing-library/issues/517

### CODE REVIEW

**Comment by trurl-master** [comment]
@eps1lon Thank you for your comment! Please check the updated version

File: docs/react-testing-library/setup.md [hunk 1]
```diff
@@ -244,6 +244,29 @@ $ app
 NODE_PATH=src/utils
 ```
 
+### Jest 24 (or lower) and defaults
+
+If you're using the Jest testing framework version 24 or lower with the default
+configuration, it's recommended to use `jest-environment-jsdom-fifteen` package
+as Jest uses a version of the jsdom environment that is not fully compatible
```
Comment (eps1lon) [inline docs/react-testing-library/setup.md]: It's not about compatibility but about missing fixes/features in earlier versions. At least this applies to other testing-library packages (e.g. `@testing-library/dom`.

Comment (trurl-master) [reply docs/react-testing-library/setup.md]: Sure. Please check the new version. If you have a better way of describing it, I'll gladly use it as my English skills is relatively limited. Thank you again for the feedback.

**Comment by kentcdodds** [comment]
@all-contributors please add @trurl-master for docs

**Pull request merged**